src/main/java/roomescape/member/controller/MemberController.java

```diff
@@ -0,0 +1,75 @@
+package roomescape.member.controller;
+
+import jakarta.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseCookie;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.member.domain.Visitor;
+import roomescape.member.dto.MemberLoginRequest;
+import roomescape.member.dto.MemberNameResponse;
+import roomescape.member.dto.MemberResponse;
+import roomescape.member.dto.MemberSignUpResponse;
+import roomescape.member.dto.MemberSignupRequest;
+import roomescape.member.service.MemberService;
+
+@RestController
+public class MemberController {
+
+    private final MemberService memberService;
+
+    public MemberController(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    @PostMapping("/members")
+    public ResponseEntity<MemberSignUpResponse> signup(@Valid @RequestBody MemberSignupRequest memberSignupRequest) {
+        memberService.signup(memberSignupRequest);
+        return ResponseEntity.created(URI.create("/login")).body(new MemberSignUpResponse("회원가입에 성공하였습니다."));
```

Comment:
응답의 세부 내용이 Controller 단에 너무 노출되어 있네요.
아래와 같이 작성하면 어떨까요?

```suggestion
        return ResponseEntity.created(URI.create("/login")).body(MemberSignUpResponse.ofSuccess());
```

Reply:
넵!! 구체적인 문자열은 controller의 관심사는 아닌 것 같습니다.
수정하였습니다.